presto-parser/src/main/java/com/facebook/presto/sql/tree/ExpressionTreeRewriter.java

```diff
@@ -555,6 +549,32 @@ public Expression visitFunctionCall(FunctionCall node, Context<C> context)
             return node;
         }
 
+        // Since OrderBy contains list of SortItems, we want to process each SortItem's key, which is an expression
+        private OrderBy rewriteOrderBy(OrderBy orderBy, Context<C> context)
```

Comment:
`OrderBy` is not an expression so cannot be added to the visitor there unless we do something hacky like returning `Object` and downcasting it depending on the context, which is probably not a good idea.